core/dbt/task/deps.py

```diff
@@ -82,9 +98,30 @@ def run(self):
                 fire_event(EmptyLine())
                 fire_event(DepsNotifyUpdatesAvailable(packages=packages_to_upgrade))
 
+    @classmethod
+    def _get_unset_profile(cls) -> UnsetProfile:
```

Comment:
Is function [`_get_rendered_profile`](https://github.com/dbt-labs/dbt-core/blob/a0ec0b6f9db74701ee701f2415ad47c8b91300ba/core/dbt/config/runtime.py#L585) still needed after adding this new function?

Reply:
I believe the commands that still depend on UnsetProfileConfig (just clean) still require it at this point. But once CleanTask is implemented with click, we should be able to rip out that method and all of UnsetProfileConfig!

Reply:
Maybe leave a note in the clean task so when @stu-k  works on it he has the infomation?